src/common/low_precision_transformations/src/mat_mul.cpp

```diff
@@ -168,7 +168,7 @@ bool MatMulTransformation::transform(TransformationContext &context, ngraph::pat
         ov::op::TemporaryReplaceOutputType(parent, deqPrecision).get(),
         ov::op::TemporaryReplaceOutputType(newMulConst, deqPrecision).get());
 
-    newMultiply->set_friendly_name(newMatMul->get_friendly_name() + "/DequantizationMultiply");
+    newMultiply->set_friendly_name(newMatMul->get_friendly_name());
```

Comment:
I think this is an incorrect change.
We should set the friendly_name from old matmul node  to this new multiply
But as I can see `updateOutput(context, newMultiply, newMatMul);` on line 176 already do it
maybe for [network_helper.cpp](https://github.com/openvinotoolkit/openvino/pull/17936/files#diff-030828c31ef1590e501e4fd5c4e58cf49e3520cd5a18de79e88fa1cadf46bfb9) we should also use updateOutput function

Reply:
Agreed. Change on this line is redundant because `updateOutput` already does such renaming. 

But with reusing `updateOutputs` in network_helper.cpp there are problems, we don't have variable `TransformationContext &context` to call it from `network_helper.cpp`. Therefore have added manual renaming to `network_helper.cpp` if there are `Results`. Please take a look.
 
Also @eshoguli can you confirm that the current solution is ok?

Reply:
`decomposeFakeQuantize` is used by some transformation. In the transformation: can you check if `updateOutput` is used after FQ decomposition?

Reply:
rewritten with `updateOutput`